src/sk_gpu_dx11.cpp

```diff
@@ -226,7 +226,8 @@ void skg_draw_begin() {
 skg_platform_data_t skg_get_platform_data() {
 	skg_platform_data_t result = {};
 	result._d3d11_device = d3d_device;
-
+	result._d3d11_current_context = GetCurrentThreadId() == d3d_main_thread ? d3d_context : d3d_deferred;
```

Comment:
This feels a little hidden, I think I'd probably prefer having this just provide two different contexts?

Reply:
Yeah, even in SK it could be good to keep them somewhat separate? I could also see SK giving an automatically selected context by default, and then also providing a way to explicitly request one or the other, buuut that could be overkill.

Reply:
It's definitely an easy add, just not sure how clear it'd be to devs to always check the type if they use the automatic one. Unless sk provides info on that d3d type check as well. Could be like an optional parameter to `void* sk_get_current_context(bool* is_main = nullptr)` or similar